extension.bzl

```diff
@@ -130,51 +130,6 @@ def _build_runner(repository_ctx):
 TELEMETRY_REGISTRY["runner"] = _build_runner
 
 
-def _repo_id(repository_ctx):
-    """Try to extract an aggregation ID (hash) from the repo context.
-
-    If there's a well known repo URL, strip user details from that and use it.
-    Otherwise use the name of the repo directory.
-    """
-
-    repo = None
-    for var in [
-        "BUILDKITE_REPO",        # Buildkite
```

Comment:
You don't think we'd want to keep this info? Would we ever have a scenario where we have the README data for local and the CI data on CI and we want to be able to link them?

Reply:
At this point I don't think it's worth it to have `id` and `id2` or something and do a migration. IDs are supposed to be for cardinality purposes only, not a huge deal if we change the strategy (or someone salts them).